mix.exs

```diff
@@ -43,7 +43,7 @@ defmodule Guardian.Mixfile do
 
   defp deps do
     [{:jose, "~> 1.8"},
-     {:phoenix, "~> 1.2.0", optional: true},
+     {:phoenix, "~> 1.2.0 or ~> 1.2-rc", optional: true},
```

Comment:
Oh you know what. `override: true` works when I put on the `phoenix` dependency. Sorry, I'm kinda new to this. But yeah, you'd think that `~> 1.2` would include `1.3-rc`, but it needs to be explicit. I used this as an example to make it work: https://github.com/phoenixframework/phoenix_live_reload/blob/master/mix.exs#L33

I guess this PR can be closed as `override: true` does work when you plug it in the proper place.